### PR TITLE
Edouard meeting template

### PR DESCRIPTION
**Pull request by edouarde1**

### CODE REVIEW

File: weekly_meeting_notes/sept_20.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# 🚀 Sept 20th 
+**Start Time** 9:30am
+
+**End Time** 10:30am
+
+**Location**
```
Comment (vnajack) [inline weekly_meeting_notes/sept_20.md:6]: can add that we had the meeting over Discord

File: weekly_meeting_notes/sept_20.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+- Logan Set up repo
+- Group selected programming exercise idea 2
+
+
+## Board Update
+
+- Exploration: Finding a string for test case
+- Task x 4 (Each feature listed above)
+
+## To Do List
```
Comment (vnajack) [inline weekly_meeting_notes/sept_20.md:77]: I appreciate having a to-do list, thanks for adding this section!

**Review by vnajack — CHANGES_REQUESTED** [review]
Great work! Thank you for taking the minutes!

The teamMeeting_template.md just needs its headers updated to what we have in the sept_20.md although it might be worth changing the naming scheme of our notes (e.g., 2022-09-20) so that the files will be in order throughout the whole school year.

File: weekly_meeting_notes/sept_20.md (hunk 2 above)
Comment (edouarde1) [reply weekly_meeting_notes/sept_20.md:77]: no problem!

File: weekly_meeting_notes/sept_20.md (hunk 1 above)
Comment (edouarde1) [reply weekly_meeting_notes/sept_20.md:6]: Added!

**Pull request closed without merging**